test/servico/cst02Test.java

```diff
@@ -64,9 +61,9 @@ public void tearDown() {
         List<AusenciaModel> ausencias = notificaçãoService.listarAusencias();        
         assertEquals(0, ausencias.size());
         
-        notificaçãoService.notificarAusencia(professor.id, "25/11/2013", "25/11/2013", "Motivo Declarado", new LinkedList<String>());
+        notificaçãoService.notificarAusencia(professor.id, "25/11/2013", "25/11/2013", "Palestra", new LinkedList<String>());
         
-        assertTrue(loginService.VerificarUsuarioESenha("Administrador", "123456"));
+        assertTrue(loginService.verificarUsuarioESenha("Administrador", "123456"));

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 83).](https://www.codacy.com/app/calebepb/substituicao/file/2738683782/issues/source?bid=3308597&fileBranchId=3353664#l66)